src/components/dialog/dialog.js

```diff
@@ -548,27 +569,6 @@ function MdDialogProvider($$interimElementProvider) {
           return angular.element(closeButton);
         }
       }
-
```

Comment:
@topherfangio  - @jelbourn asked me about the `<p>` wrappers and I could not recall why I did that originally. LOL